src/main/java/gift/controller/WishlistController.java

```diff
@@ -37,8 +39,10 @@ public ResponseEntity<WishResponse> create(@Valid @RequestBody WishRequest reque
     }
 
     @GetMapping()
-    public ResponseEntity<List<WishResponse>> findAll(@LoginMember LoginMemberDto memberDto) {
-        List<WishResponse> wishResponses = wishlistService.findAllByMemberId(memberDto);
+    public ResponseEntity<Page<WishResponse>> findAll(
+        @RequestParam(value = "page", defaultValue = "0") int page,
+        @LoginMember LoginMemberDto memberDto) {
+        Page<WishResponse> wishResponses = wishlistService.findAllByMemberId(page, memberDto);
         return ResponseEntity.status(HttpStatus.OK).body(wishResponses);
```

Comment:
page만 받는 것도 가능하지만 아래와 같이, Pageable을 사용하면 page 크기도 설정할 수 있습니다~
```suggestion
    public ResponseEntity<Page<WishResponse>> findAll(
        Pageable pageable
        @LoginMember LoginMemberDto memberDto) {
        Page<WishResponse> wishResponses = wishlistService.findAllByMemberId(pageable, memberDto);
        return ResponseEntity.status(HttpStatus.OK).body(wishResponses);
```